extensions/pkg/terraformer/state.go

```diff
@@ -19,7 +19,9 @@ import (
 	"encoding/json"
 	"fmt"
 
+	"github.com/gardener/gardener/pkg/controllerutils"
```

Comment:
`github.com/gardener/gardener/pkg/controllerutils.HasFinalizer` is already deprecated in favour of `sigs.k8s.io/controller-runtime/pkg/controller/controllerutil.ContainsFinalizer`

```suggestion
	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
```

Reply:
Done